iroh-net/src/relay/server.rs

```diff
@@ -1,903 +1,1042 @@
-//! based on tailscale/derp/derp_server.go
-use std::collections::HashMap;
+//! A full-fledged iroh-relay server.
```

Comment:
Again, the diff here ended up bad. I think it may actually be due to the fact that I've moved this module somewhere else while simultaneously moving something else to here :/